R/indicator_backlog.R

```diff
@@ -1,28 +1,47 @@
-available_indicators <- function(indicator = NULL){
```

Comment:
we got a merge issue in this function that should be resolved before merging to main

Reply:
thank you for the super-fast response. I will soon update the changes